yarn-project/archiver/src/archiver/archiver.test.ts

```diff
@@ -116,31 +77,22 @@ describe('Archiver', () => {
     latestBlockNum = await archiver.getBlockNumber();
     expect(latestBlockNum).toEqual(3);
 
-    // New L1 to L2 messages
+    // L1 to L2 messages
     {
       // Checks that I get correct amount of sequenced new messages for L2 blocks 1 and 2
-      let newL1ToL2Messages = await archiver.getNewL1ToL2Messages(1n);
-      expect(newL1ToL2Messages.length).toEqual(2);
+      let l1ToL2Messages = await archiver.getL1ToL2Messages(1n);
+      expect(l1ToL2Messages.length).toEqual(2);
 
-      newL1ToL2Messages = await archiver.getNewL1ToL2Messages(2n);
-      expect(newL1ToL2Messages.length).toEqual(3);
+      l1ToL2Messages = await archiver.getL1ToL2Messages(2n);
+      expect(l1ToL2Messages.length).toEqual(3);
 
       // Check that I cannot get messages for block 3 because there is a message gap (message with index 0 was not
```

Comment:
Can you expand a bit on this one, I'm not fully following it.

Reply:
Clarified the comment in 3381334ec4c71e8a0def3a6ac69435fc417d1c58